src/amo/components/Home/styles.scss

```diff
@@ -94,6 +117,12 @@
 .Home-SubjectShelf-link {
   font-size: $font-size-m-smaller;
   text-decoration: none;
+  display: flex;
+  align-items: center;
```

Comment:
@hritvi , for fixing errors you need to arrange css properties in alphabetical order . You can check the error here https://travis-ci.org/mozilla/addons-frontend/jobs/425625754#L517-L536